### ISSUE TITLE
[Bug Report][3.3.7] Navigation in VSelect (arrow keys) does not work with NVDA enabled

### ISSUE THREAD

**Issue by paul-sudo** [vuetifyjs/vuetify#17767]
### Environment
**Vuetify Version:** 3.3.7
**Vue Version:** 3.3.4
**Browsers:** Google Chrome
**OS:** Windows

### Steps to reproduce
- Install and open NVDA.
- Open playground.
- Try to navigate in the select input using keyboard.

### Expected Behavior
Navigation works as when NVDA is not enabled.

### Actual Behavior
Stops at the first element, navigation just doesn't work.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqlUU1LAzEQ/StjLqvQ3RwUhFJFEdS7goduD3F3WqP5YpJWpfS/O8m2WOtF8DZ5M/PevJfpWlyH0KyWKMZiktAGoxJetg5gsqpVCKUsj867pLRD2kIFjGiwSzsAwKhnNBeteBhw8d0Za2aP3JpWN8rouSenVTWC6sYbT6r3ub7lUvcFvkNPi2HiET9UzMXTp7faLarZN/HlRO6u2J3KwM9bMzJYmcg9i/yMHemQIGJaDk61DZ4SrIFwDhuYk7dQcTpVHgdg3pjAxgVc5Inj6h6N8fDkyfRH1UkWGCiZTIzEwFZbFZrX6B1HvM4s7bYRWzGGgmSMVfK7FS8phTiWsusdr/Vo9Ioah0m6YOUVj0lauqQt1r23V6fNaXMmex3TPtxgtPUz+feIxCStGO3JJD3//IPUdrIonA8KW6jmb45Z4he1ZL0VUk3oeiSkvzo6WNt3ddD65Syrb1q34byNdm/xIOoulpinuyP/ZbmQZaZZFtyMRLmTlcuCmH0BlQUXtA==)
